All EIPIP Meetings/Meeting 007.md

```diff
@@ -0,0 +1,527 @@
+
+And that's already kind of half of what happens, right? Half the conversation happens in the GitHub PR for the EIP, and the only part we're missing there is, like, four clients adding, like, a plus one that, okay, this should be accepted. And that doesn't seem that hard to do async. But I agree that the more fuzzy scheduling is hard.
+
+**Edson Ayllon**
+One comment on that. There has been a need expressed in the survey for clarity on where the EIP process ends and where the hard fork coordination begins. Or at least how those two are separate, clarifying that part.
+
+**Pooja Ranjan**
+I would agree, for the most part, with James as well as Tim. Right now, the process of EIP and the hard fork is not actually separated, but in my opinion, the EthMagicians is a very good forum. We can go ahead and do that in addition to James's idea of collecting information from individual clients, we can also start referring to the point of discussion. And if we make it a preference of having EthMagicians instead of the PR of that, we would be having collective information in one place. And even if it is async, we're also getting async information during the AllCoreDev call.
+
+So there is basically a fine line between the hard fork process and the EIP process itself. I tried to document that sometime back in one of the EIPs - it must have been moved somewhere else. But yeah, I find that this may be a good way forward in which people will be getting accurate information even about all the clients and where they stand at any point in time.
+
+**Hudson Jameson**
+Okay, so we've kind of agreed that the process of hard fork management and EIP development should be separate, or implementation should be separate, right?
+
+**James Hancock**
+The EIP standardization and the network upgrade deployment should be separate.
+
+**Hudson Jameson**
+OK, so we're all there. We're saying that there might need to be new tools in order to make that happen. We're also saying there might need to be, or that there is value in having some type of asynchronous coordination beyond the AllCoreDevs Gitter chat for people who are in different time zones, and that there might be tools we need to bring up to do that. I think I got that all right. Is there anything I got wrong?
+
+**Pooja Ranjan**
+We already have existing tools. We just need to enforce the acceptance and implementation of that. We have a discussion port on the implementation. In my understanding, most of the EIPs are there, and they are referred to be as a point of discussion. But sometimes what happens is the discussion shifts to the pull request. And there we lost the track. So if we can make our standard process a standard place, that this is it, and this is where you need to go, and we refer the link in AllCoreDev meetings that might bring all the information to one place.
```

Comment:
Probably, we already have existing tools. We just need to enforce the acceptance and implementation of that. We have a discussion forum, the Eth Magicians. In my understanding, most of the EIPs are there, and they are referred to be as a point of discussion. But sometimes what happens, the discussion shifts to the pull request. And there we lost the track. So if we can make a standard process, a standard place, that this is it, and this is where you need to go, and we refer the link in AllCoreDev meetings, that might bring all the information to one place.